tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -337,6 +359,44 @@ public static ExplorationTestDescription GetExplorationTestDescription(Explorati
                 TestsToIgnore = new[] { "Loading assembly metadata works" },
                 SupportedFrameworks = new[] { TargetFramework.NET461 },
             },
+            ExplorationTestName.RestSharp => new ExplorationTestDescription()
+            {
+                Name = ExplorationTestName.RestSharp,
+                GitRepositoryUrl = "https://github.com/restsharp/RestSharp.git",
+                GitRepositoryTag = "107.0.3",
+                IsGitShallowCloneSupported = true,
+                PathToUnitTestProject = "test/RestSharp.Tests",
+                SupportedFrameworks = new[] { TargetFramework.NET6_0 },
+            },
+            ExplorationTestName.serilog => new ExplorationTestDescription()
+            {
+                Name = ExplorationTestName.serilog,
+                GitRepositoryUrl = "https://github.com/serilog/serilog.git",
+                GitRepositoryTag = "v2.10.0",
+                IsGitShallowCloneSupported = true,
+                PathToUnitTestProject = "test/Serilog.Tests",
+                TestsToIgnore = new[] { "DisconnectRemoteObjectsAfterCrossDomainCallsOnDispose" },
+                SupportedFrameworks = new[] { TargetFramework.NETCOREAPP3_1 },
+            },
+            ExplorationTestName.polly => new ExplorationTestDescription()
+            {
+                Name = ExplorationTestName.polly,
+                GitRepositoryUrl = "https://github.com/app-vnext/polly.git",
+                GitRepositoryTag = "7.2.2+9",
+                IsGitShallowCloneSupported = true,
+                PathToUnitTestProject = "src/Polly.Specs",
+                SupportedFrameworks = new[] { TargetFramework.NETCOREAPP3_1, TargetFramework.NET5_0, TargetFramework.NET461 },
+            },
+            ExplorationTestName.automapper => new ExplorationTestDescription()
+            {
+                Name = ExplorationTestName.automapper,
+                GitRepositoryUrl = "https://github.com/automapper/automapper.git",
+                GitRepositoryTag = "v11.0.0",
+                IsGitShallowCloneSupported = true,
+                PathToUnitTestProject = "src/UnitTests",
+                SupportedFrameworks = new[] { TargetFramework.NET6_0 },
+                IsWindowsOnly = true,
```

Comment:
Just a nitpick, but may be worth to introduce something like `SupportoedOS` enum?

Reply:
Good idea, replaced `IsWindowsOnly` with `SupportedOSPlatforms` using the library type `System.Runtime.InteropServices.OSPlatform`. I'll go ahead and merge this change for now so I can finally get it merged, but we can always iterate on this idea later